benchmark/fs/bench-rmdirSync.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const path = require('path');
+const tmpdir = require('../../test/common/tmpdir');
+tmpdir.refresh();
+
+const testFiles = fs.readdirSync('test', { withFileTypes: true })
+  .filter((f) => f.isDirectory())
```

Comment:
This literally deletes existing folders. Can you make sure we don't delete any useful folders?